File: Java Script - Introduction to Programming/8.2. Exam Preparation - Part II/01. Distance.js
function distance([...arg]) {
  let [kmSpeed, firstTimeMin, secondTimeMin, threeTimeMInt] = [...arg.map(Number)]

  let cap = kmSpeed * firstTimeMin / 60
  let cap2 = (kmSpeed + kmSpeed * 0.10) * (secondTimeMin / 60)
  let cap3 = ((kmSpeed + kmSpeed * 0.10) * 0.95) * (threeTimeMInt / 60)

  return (cap + cap2 + cap3).toFixed(2)

}

// console.log(distance([' 140', '112', '75 ', '190 ']))

File: Pyrhon - Introduction to Programming/8.2. Exam Preparation - Part II/01. Distance.py
km_speed, first_time_min, second_time_min, three_time_mInt = [int(input()) for _ in range(4)]

cap = km_speed * first_time_min / 60
cap2 = (km_speed + km_speed * 0.10) * (second_time_min / 60)
cap3 = ((km_speed + km_speed * 0.10) * 0.95) * (three_time_mInt / 60)

print(f'{(cap + cap2 + cap3):.2f}')


'''
90
60
70
80
#330.90
'''
